app/assets/stylesheets/nav.scss

```diff
@@ -10,34 +16,63 @@ $button-color: #1E90FF;
   list-style-type: none;
 }
 
+.top-bar {
+  background-color: #3f3f3f;
+}
+h3{
+  font-family: $font-sans;
+}
+h2{
+  color: $white;
+}
+body{
+  background-color: $background;
+  font-family: $font-sans;
+}
+body.home{
+  background-size: relative;
+  background-image: url("http://jarques.me/img/potd_sept_30.jpg");
+}
 .container {
   margin-left: 10%;
   margin-right: 5%;
-
 }
 
 .inline {
   display: inline-block;
-
   input {
     width:280px;
+    border-radius: 5%;
   }
   label {
       color: orange;
       font-weight: bold;
+      span{
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented 4 spaces, but was indented 6 spaces<br>Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space